pkg/webui/console/containers/device-onboarding-form/device-provisioning-form-section/index.js

```diff
@@ -38,62 +41,127 @@ import DeviceRegistrationFormSection, {
   initialValues as registrationInitialValues,
 } from './device-registration-form-section'
 
+const msg = defineMessages({
+  continue:
+    'To continue, please enter the JoinEUI of the end device so we can determine onboarding options',
+  confirm: 'Confirm',
+  reset: 'Reset',
+  registration: '✔ This end device can be registered on the network',
+  claiming: '✔ This end device can be claimed from its current owner',
+})
+
 const initialValues = merge(
   {
+    authenticated_identifiers: {
+      join_eui: '',
+    },
     ids: {
       join_eui: '',
     },
+    _claim: undefined,
   },
   claimingInitialValues,
   registrationInitialValues,
 )
 
-const DeviceTypeFormSection = () => {
-  const [isClaiming, setIsClaiming] = useState(undefined)
+// Save EUI in both fields.
+const joinEuiEncoder = value => ({
+  ids: { join_eui: value },
+  authenticated_identifiers: {
+    join_eui: value,
+  },
+})
```

Comment:
```
const joinEuiEncoder = value => ({
  ids: { ...value.ids, join_eui: value },
  authenticated_identifiers: {
    ...value.authenticated_identifiers,
    join_eui: value,
  },
})
```
Otherwise, you lose the existing values there which will cause issues when submitting.

Reply:
Please disregard this change actually. It should work as you put it. The issue is that the base branched is not rebased on latest master.